docs/DeveloperGuide.md

```diff
@@ -283,33 +283,97 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
-
-**Use case: Delete a person**
-
-**MSS**
-
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+(For all use cases below, the **System** is the `UNion Application` and the **Actor** is the `user`, unless specified otherwise)
+
+**System: UNion Application (UNa)**\
+**Use case: UC01 - Add a new folder**\
+**Actor: User**\
+**Guarantees:**
+   * A new folder with the specified name is created as part of the application.
+
+**MSS:**
+   1. User keys in folder name after `mkdir` keyword.
```

Comment:
> A use case describes an interaction between the user and the system for a specific functionality of the system.

the format of MSS seems to be

1. user does something
2. system does something
3. user does something
4. system does something

what should the system do after user requests to create a new folder?

same for all subsequent use cases